modules/ipfs-cpinner-client/README.md

```diff
@@ -33,17 +35,21 @@ const storage: ClientKeyValueStorage = myCustomStorage
 const serviceDid = 'did:ethr:rsk:0x123456789....abc'
 const address = '0xabcdef....123' // user's address
 const did = `did:ethr:rsk:${address}`
-const rpcPersonalSign = (data: string) => window.ethereum.request({ method: 'personal_sign', params: [address, data] }) // this is an example with Metamask
 
-const client = new DataVaultWebClient({ serviceUrl, did, rpcPersonalSign, serviceDid, storage })
+// these are examples with Metamask
+const rpcPersonalSign = (data: string) => window.ethereum.request({ method: 'personal_sign', params: [address, data] })
+const decrypt = (hexCypher: string) => window.ethereum.request({ method: 'eth_decrypt', params: [hexCypher, address] })
+const getEncryptionPublicKey = () => window.ethereum.request.request({ method: 'eth_getEncryptionPublicKey', params: [address] })
```

Comment:
We can get it once. It should be always the same for the same address

Reply:
https://www.freecodecamp.org/news/understanding-memoize-in-javascript-51d07d19430e/